generate_client.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# Generate golang client sdk from OpenAPI 3.0.0 spec
```

Comment:
Hmm, this is sub-optimal. What would it take to add the functionality needed by the webapp to the community_python_sdk instead of accessing the API directly using the swagger generated client?

Reply:
Fixed in https://github.com/kentik/sla_dashboard_webapp/pull/2